src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/model/ccd/ScannedDocument.java

```diff
@@ -33,25 +32,4 @@ public ScannedDocument(
         this.url = url;
         this.exceptionReference = exceptionReference;
     }
```

Comment:
these are needed in functional tests

Reply:
SupplementaryEvidenceTest (functional test) is disabled at the moment, but they will start failing with this change

Reply:
Fair enough. Closing.